kong/dynamic_hook/init.lua

```diff
@@ -110,28 +226,27 @@ function _M.run_hooks(ctx, group_name, hook_name, ...)
 
   local ok, err = pcall(handler, ...)
```

Comment:
`...` is NYI here, can we replace it with `a0, a1, a2, a3` and set up safeguards to handle more than N arguments?

```
---- TRACE 1434 start 1422/0 @./kong/dynamic_hook/init.lua:194
0001  UGET     1   0      ; ALWAYS_ENABLED_GROUPS       (@./kong/dynamic_hook/init.lua:194)
0002  TGETV    1   1   0       (@./kong/dynamic_hook/init.lua:194)
0003  ISF          1       (@./kong/dynamic_hook/init.lua:194)
0004  JMP      2 => 0007
0005  KPRI     1   2       (@./kong/dynamic_hook/init.lua:195)
0006  RET1     1   2       (@./kong/dynamic_hook/init.lua:195)
0005  IST          3       (@./kong/dynamic_hook/init.lua:213)
0006  JMP      3 => 0008
0008  UGET     3   1      ; NON_FUNCTION_HOOKS       (@./kong/dynamic_hook/init.lua:217)
0009  TGETV    3   3   1       (@./kong/dynamic_hook/init.lua:217)
0010  IST          3       (@./kong/dynamic_hook/init.lua:218)
0011  JMP      4 => 0013
0013  TGETV    4   3   2       (@./kong/dynamic_hook/init.lua:222)
0014  IST          4       (@./kong/dynamic_hook/init.lua:223)
0015  JMP      5 => 0017
0017  UGET     5   2      ; pcall       (@./kong/dynamic_hook/init.lua:227)
0018  MOV      7   4       (@./kong/dynamic_hook/init.lua:227)
0019  VARG     8   0   3       (@./kong/dynamic_hook/init.lua:227)
---- TRACE 1434 abort @./kong/dynamic_hook/init.lua:227 -- NYI: bytecode 71
```

Reply:
@ADD-SP I did a small modification to this, not sure if it is effective.